src/components/Differential/ScatterPlotDiv.jsx

```diff
@@ -324,41 +358,25 @@ class ScatterPlotDiv extends Component {
       xAxisLabel !== null &&
       yAxisLabel !== null
     ) {
-      const xAxisTransformBox = allowXTransformation ? (
+      const xAxisTransformBox = (
         <Form.Field
           control={Checkbox}
           name="xTransformationCheckbox"
           className="VolcanoTransformationCheckbox"
           checked={doXAxisTransformation}
+          disabled={!allowXTransformation}
```

Comment:
ternary not necessary on the entire component, when it can just be done on disabled